api/src/main/java/org/hawkular/apm/api/model/trace/Node.java

```diff
@@ -324,16 +348,18 @@ public void addInteractionId(String id) {
      * @param scope The scope
      * @return The subset of correlation ids that are associated with the scope
      */
-    public List<CorrelationIdentifier> getCorrelationIds(Scope scope) {
+    public List<CorrelationIdentifier> findCorrelationIds(Scope... scope) {
```

Comment:
actually set is a good idea otherwise this method can return multiple ids for the same scope. Or just create a set from the input parameter.